manifest.yml

```diff
@@ -0,0 +1,34 @@
+name : "AWS_IoT_Device_SDK_for_Embedded_C"
+version: "202009.00"
+description: |- 
+  "The AWS IoT Device SDK for Embedded C is a collection of C source
+  files that can be used in embedded applications to securely connect 
+  to AWS IoT Core and interact with other AWS IoT services on AWS Cloud. \n
+  \n
+  See dependencies for included libraries."
+dependencies:
+  - name: "coreMQTT"
+    version: "1.0.0"
+      type: "git"
+      url: "https://github.com/FreeRTOS/coreMQTT"
+  - name: "coreHTTP"
```

Comment:
http is not getting released now.

Reply:
This pull request is for master, the one for release candidate does not have HTTP.